packages/repository/src/relations/relation.types.ts

```diff
@@ -117,7 +117,7 @@ export interface BelongsToDefinition extends RelationDefinitionBase {
   /*
    * The foreign key in the source model, e.g. Order#customerId.
    */
-  keyFrom: string;
+  keyFrom?: string;
```

Comment:
I have mixed feelings about this change. On one hand, I think it allows us to simplify `belongsTo` method to accept `definition: Omit<BelongsToDefinition, 'name' | 'type' | 'targetsMany'>` and is kind of consistent with other relation types that have `keyFrom` optional too.

On the other hand, I am not able to fully asses ramifications of this change, I have no idea what it could break.

On the third hand :) Since this is backwards compatible and you have code to handle the case where `keyFrom` is not provided, I think it should be fine.